src/skills-builder/skills-builder-modal/view-results/CarouselStack.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CardCarousel } from '@edx/paragon';
+import RecommendationCard from './RecommendationCard';
+
+const CarouselStack = ({ selectedRecommendations }) => {
+  const { name: jobName, recommendations } = selectedRecommendations;
+  const lobNames = Object.keys(recommendations);
+
+  const normalizeLOBName = (lobString) => {
+    const newLobString = lobString.split('_').map(word => word[0].toUpperCase() + word.slice(1).toLowerCase());
+    return newLobString.join(' ');
+  };
```

Comment:
Nit: could we add a comment as to why this is needed/important?

Reply:
Yes, good call. This function takes in a string that is either a single word (such as "course") or two words separated by an underscore ("boot_camp"). The string is then returned with the first letter of each word capitalized and any underscored replaced as spaces. I'll find a way to fit this in neatly as a comment 👍